affiliatewp-affiliate-area-shortcodes.php

```diff
@@ -27,8 +27,8 @@
 	exit;
 }
 
-if ( ! class_exists( 'AffiliateWP_Requirements_Check' ) ) {
-	require_once dirname( __FILE__ ) . '/includes/lib/affwp/class-affiliatewp-requirements-check.php';
+if ( ! class_exists( 'AffiliateWP_Requirements_Check_v1_1' ) ) {
+	require_once dirname( __FILE__ ) . '/includes/lib/affwp/class-affiliatewp-requirements-check_v1_1.php';
```

Comment:
Looks like the file path is incorrect. Should be hyphens instead of underscores.

```suggestion
	require_once dirname( __FILE__ ) . '/includes/lib/affwp/class-affiliatewp-requirements-check-v1-1.php';
```